docs/experiments/experiment1.md

```diff
@@ -1,9 +1,104 @@
+
+The following is an example of a request that searches with match query. It operates on the query string in the 'text' field of the medline corpus:
+
+```
+"request": {
+                "query": {
+                    "match": {
+                        "text": "palliation (temporary improvement) of cancer patients by using drugs, x-ray, surgery."
+                    }
+                }
+            }
+        },
+```
+
+## Simple query string operator
+
+The [simple query string](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html) operator uses a parser with a limited fault-tolerant syntax to search for the given query string. It splits and analyzes the query terms separately for the search before returning the relevant documents. It ignores invalid parts of the query string. 
+
+Here is an example request with the simple query string operator searching for the query in the 'text' and 'title' fields of the medline corpus documents:
+
+```
+"request": {
+                "query": {
+                    "simple_query_string": {
+                        "fields": [
+                            "title",
+                            "text"
+                        ],
+                        "query": "palliation (temporary improvement) of cancer patients by using drugs, x-ray, surgery."
+                    }
+                }
+            }
+        },
+```
+
+## Stemming
+
+Elasticsearch provides two build-in stemming methods. First of, the dictionary stemming replaces unstemmed words with stemmed variants from a provided dictionary. The Elasticsearch build-in conterpart is called [hunspell token filter](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-hunspell-tokenfilter.html). Second of, is the algorithmic stemming. It applies a series of rules to each word to reduce them, called [stemmer token filter](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-stemmer-tokenfilter.html) in Elasticsearch. 
+([Click here to read more about stemming in general](https://pragmalingu.de/docs/guides/basic-definitions#stemming)) 
+
+The *stemmer token filter* supports different languages and, since our corpora were all in english, we utilized the english stemmer for our experiment.
+We implemented the stemming analyzer:
+
+```
+stemming_analyser = {
+    "filter" : {
+        "eng_stemmer" : {
+        "type" : "stemmer",
+        "name" : "english"
+        }
+    },
+    "analyzer" : {
+        "default" : {
+            "tokenizer" : "standard",
+            "filter" : ["lowercase", "eng_stemmer"]
+        }
+    }
+}
+```
+
+And incorporated it into the settings for the body of the documents of each corpus:
+
+```
+settings = {
+    "settings": {
+        "number_of_shards": 1,
+        "number_of_replicas": 0,
+        "analysis": stemming_analyser
```

Comment:
the stemming_analyzer string is missing quotes

Reply:
It' a variable where Dario safed the analyser that's why it's not in quotes but it gets resolved by also adding python

Reply:
Ah ok sry, i didn't see that 😅